app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    temp = command.split(" ")
+    if temp[0] != "cp" or temp[1] == temp[2]:
```

Comment:
u can unpack this variables using split like this: `arg1, arg2, arg3 = some_string.split()`
also no need to return 0 here, u can just use backward condition

Reply:
Fixed)